src/main/cli/settings.c

```diff
@@ -165,7 +165,7 @@ const char * const lookupTableOffOn[] = {
 };
 
 static const char * const lookupTableCrashRecovery[] = {
-    "OFF", "ON" ,"BEEP", "DISARM"
+    "OFF", "ON" ,"BEEP", "DISARM", "LEVEL"
```

Comment:
Potential confusion here. Crash recovery already has a leveling aspect. That's the point of it's operation - to get the craft under control and back to level before returning control.